babel.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = function (api) {
           browsers: ['Chrome >= 60', 'Firefox >= 44', 'Safari >= 10', 'Edge >= 17'],
         },
         useBuiltIns: 'entry',
-        corejs: 2,
+        corejs: 3,
```

Comment:
`@babel/polyfill` was using inside `corejs` (in version 2).
This upgrade was necessary because Babel recommend not using `@babel/polyfil` [anymore](https://babeljs.io/docs/en/babel-polyfill).
So I update core-js version to 3 in this MR.